### DISCUSSION TITLE
propose changing companies/weberpdemo to companies/default

### DISCUSSION THREAD

**Discussion by dalers** [timschofield/webERP#266] [Development Discussion & Specification]
It seems the companies/weberpdemo directory now exists only for copying when a new company is created (it also already exists if a "weberpdemo" company happens to be created), but the directory name is misleading and can be confusing when a weberpdemo company (i.e. database) has not been created.

For clarity and to avoid any confusion, I suggest companies/weberpdemo be renamed to companies/default (or something more suitable).

I expect the change to require changes in at least the installer scripts if not others.

I also suggest .gitignore should include _all_ companies/ subdirectories _except_ for default/ 

Any thoughts?

**Comment by timschofield** [comment]
I am fine with this changing the directory name Dale, I think this goes back over 20 years to the projects start. Only thing to remember is that the directory name is also the databases name, so this needs changing everywhere (config.dist.php, various place in the installer and in the Companies.php file). Sorry for being a bit vague there, would need to physically check through to get all.

Tim

**Comment by dalers** [comment]
I had expected there to be many places to change, and yes there are, but mostly documentation and translations iiuc.

I used NetBeans[1] to search the project for "weberpdemo" - 344 matches of weberpdemo in 90 files.

Skimming through though, 90% of the matches are in messages.po files (translations?) and the user manual. The remainder are made up of config.distrib.php, page_6.php of the installer, Companies.php, RecurringSalesOrdersProcess.php, api_php.php, the SQL files, and the make release shell script and log truncate shell script.

![image](https://github.com/user-attachments/assets/2e124a38-e8b8-4217-86a2-09e4c7a366e6)

[1] Fwiw my dev setup is becomming NetBeans on Windows 10 with XAMPP PHP 8.1, AdminerEvo dbms dashboard, and GitExtensions or git bash for repo management. I'll scripting loading/re-loading a test database.

**Reply by dalers** [reply]
P.S. I won't be doing this any time soon. I'll flag this in my personal todo list but won't add to the open Issues.

**Comment by timschofield** [comment]
OK, I do think you're right, but maybe not top priority.

Thanks
Tim
